### PR TITLE
Fix the temporary state revert before block finalization

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Update new final frontend db with tx pool
- Guide it with UPDATE_NEW_FINAL_FRONT_DB_WITH_TX_POOL node param
- Update deploy scripts to set UPDATE_NEW_FINAL_FRONT_DB_WITH_TX_POOL=false for node 0 to 4
- Add package scripts commands for running cloud test run 

Background:
- This PR fixes the issue of the temporary state revert before block finalization 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/958

### CODE REVIEW

File: node/index.js [hunk 1]
```diff
@@ -907,6 +908,39 @@ class BlockchainNode {
     return 0; // Successfully merged
   }
 
+  executeAndGetValidTransactions(
+      longestNotarizedChain, blockNumber, blockTime, tempDb, isExecutionOnly = false) {
```
Comment (liayoo) [inline node/index.js:912]: minor) could you update the variable name since it's no longer just tempDb that can be passed to?

File: package.json [hunk 2]
```diff
@@ -14,12 +14,14 @@
   },
   "scripts": {
     "client": "UNSAFE_PRIVATE_KEY=b22c95ffc4a5c096f7d7d0487ba963ce6ac945bdc91c79b64ce209de289bec96 DEBUG=true node ./client/index.js",
+    "deploy_test": "bash deploy_test_gcp.sh $USER all",
+    "cat_log_test": "bash deploy_test_gcp.sh $USER all --cat-log",
```
Comment (liayoo) [inline package.json:18]: yay! 🙌

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

File: node/index.js (hunk 1 above)
Comment (platfowner) [reply node/index.js]: Just done. Thanks!

**Comment by platfowner** [comment]
Thanks for the review. Now merging.. @liayoo

**Pull request merged**